test/concurrent/ThreadTest.ooc

```diff
@@ -45,15 +45,17 @@ ThreadTest: class extends Fixture {
 				value set(value get() + 1)
 				Thread yield()
 				Time sleepMilli(1)
+				if (Thread currentThread() isCanceled())
```

Comment:
I understand. I just wanted to avoid hard-coding it or changing the base `Thread` class too much, because I feel this can be moved to a separate reusable abstraction layer (merging with already existing Tasks from ThreadPool would be ideal).